blockly-core/core/utils/variables.js

```diff
@@ -216,3 +220,27 @@ Blockly.Variables.generateUniqueName = function() {
   }
   return newName;
 };
+
+Blockly.Variables.generateUniqueNameFromBase_ = function(baseName) {
```

Comment:
Could use a short description of expected input / one example (e.g., given variable1, would return variable2)